migrating_from_ocp_3_to_4/installing-3-4.adoc

```diff
@@ -33,15 +33,15 @@ If you use a proxy server with an internally hosted replication repository, you
 
 The following storage providers are supported:
 
-* Multi-Cloud Object Gateway (MCG)
-* Amazon Web Services (AWS) S3
-* Google Cloud Platform (GCP)
-* Microsoft Azure Blob
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-mcg_installing-3-4[Multicloud Object Gateway]
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-aws-s3_installing-3-4[Amazon Web Services S3]
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-gcp_installing-3-4[Google Cloud Platform]
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-azure_installing-3-4[]Microsoft Azure Blob
```

Comment:
The ending backtick should be after the text (xref isn't rendering properly)

Reply:
Oops-- sorry, missed that!